pkgs/by-name/ne/neocities-cli/package.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, bundlerApp
+, bundlerUpdateScript
+, installShellFiles
+}:
+
+let
+  ttBundlerApp = bundlerApp rec {
+    pname = "neocities";
+    gemdir = ./.;
+    exes = [ pname ];
+
+    passthru.updateScript = bundlerUpdateScript "neocities";
+  };
+in
+
+stdenv.mkDerivation rec {
+  name = "neocities";
+  dontUnpack = true;
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${ttBundlerApp}/bin/${name} $out/bin
+  '';
+
+  meta = with lib; {
+    description = "The Neocities Gem - A CLI and library for using the Neocities web site API.";
+    homepage = "https://github.com/neocities/neocities-ruby";
+    mainProgram = name;
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> Always prefer using a hardcoded string (don't use pname, for example).

Reply:
I changed this based on @Sigmanificient message, shouldn't have. I'm going to reverse it

Reply:
no worries ! nixpkgs is a huge project, lots of things to learn

Reply:
:<